Notes/ViewModels/ContentViewModel.swift

```diff
@@ -12,20 +12,48 @@ class ContentViewModel: ObservableObject {
     @Published var usernameInput: String = ""
     @Published var passwordInput: String = ""
     @Published var errorMessage = ""
+    @Published var usernameInvalid = false
+    @Published var passwordInvalid = false
     @Published var isLinkActive = false
+<<<<<<< HEAD:Notes/ContentViewModel.swift
```

Comment:
you may need to resolve the conflict with the base branch so that there will be no added line like this one.